.github/workflows/add-reviewer-monitoring.yaml

```diff
@@ -0,0 +1,26 @@
+name: Add reviewer on monitoring PRs
+
+on:
+  pull_request:
+    types: [opened]
+    paths:
+      - internal/operands/metrics/*
+      - internal/common/resource.go
+      - docs/metrics.md
+      - tests/metrics_test_utils.go
+      - tools/metricsdocs/*
+
+jobs:
+  add_reviewer:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
```

Comment:
Do we really need to checkout the code first?

Reply:
Yes, without checkout the `gh` command does not work properly.